### PR TITLE
H1413 custom thanks message

### LINKED ISSUE #1413: Special Donation Thank You pop up for Commons Stack Campaign
What do you think about making a custom Thank You pop up for the Commons Stack... let me think about the exact copy for this..

https://beta.giveth.io/campaigns/5d110631994eac53adc8f21e/donate

### PR DESCRIPTION
**Pull request by horus2021**
Fix #1413 Special Donation Thank You pop up for Commons Stack Campaign

### CODE REVIEW

File: src/components/DonateButton.jsx [hunk 1]
```diff
@@ -1080,23 +1080,44 @@ export default class Root extends React.PureComponent {
 
   afterSuccessfulDonate() {
     const { donateToDefaultDac } = this.state;
+    const { customThanksMessage } = this.props.model;
```
Comment (aminlatifi) [inline src/components/DonateButton.jsx:1083]: `customThanksMessage` would contain the message. In this case "Thank you for your donation to the Commons Stack! If you...."

Comment (horus2021) [reply src/components/DonateButton.jsx:1083]: What is the value of customThanksMessage in the case of normal ThanksMessage?
null or No??

I want to write a condition on it?

Comment (horus2021) [reply src/components/DonateButton.jsx:1083]: I expect to fetch the following content from feathers. Please put it in MongoDB.


"Thank you for your donation to the Commons Stack! If you would like to receive <a href='https://medium.com/commonsstack/cstk-the-token-of-the-commons-stack-trusted-seed-931978625c61'>CSTK</a> tokens, please <a href= 'mailto:info@commonsstack.org?subject=I just donated to Commons Stack!' > contact us</a> and we'll send you all the details!"

Comment (horus2021) [reply src/components/DonateButton.jsx:1083]: @aminlatifi  check my new commit

**Comment by aminlatifi** [comment]
@horus2021 @GriffGreen We should revise the flow! Currently after user donate:
1. If he has donated to a project other than default DAC, he will see `Would you like to support Giveth as well?`.
2. If he has donated to default DAC the user profile is checked and ask him to fill if he has not yet!

I think scenario is incomplete and doesn't cover all cases!

**Comment by aminlatifi** [comment]
@horus2021 Fix conflict first (merge latest develop branch into your branch)

File: src/components/DonateButton.jsx [hunk 2]
```diff
@@ -1082,23 +1082,42 @@ export default class Root extends React.PureComponent {
...
+        if (customThanksMessage !== '') {
+          // Custom Thanks for Commons Stack
+          React.swal({
+            title: 'Thank you!',
+            content: el,
+            icon: 'success',
+            buttons: ['Ok!'],
+          }).then(result => {
+            if (result) {
+              this.defaultDacDonateButton.current.openDialog();
```
Comment (aminlatifi) [inline src/components/DonateButton.jsx:1102]: It's different than what was mentioned here: https://github.com/Giveth/giveth-dapp/issues/1413#issuecomment-714582026

**Review by aminlatifi — CHANGES_REQUESTED** [review]
Test your code on all conditions and make sure the logic is correct before pushing

**Pull request closed without merging**